docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,484 @@
+    }
+  },
+  "analyzed_fields": {
+    "includes": [],
+    "excludes": [    <6>
+      "Cancelled",
+      "FlightDelayMin",
+      "FlightDelayType"
+    ]
+  },
+  "model_memory_limit": "100mb" <7>
+}
+--------------------------------------------------
+// TEST[skip:setup kibana sample data]
+
+<1> The source index to analyze.
+<2> This query removes erroneous data from the analysis to improve its quality.
+<3> The index that will contain the results of the analysis; it will consist of 
+a copy of the source index data where each document is annotated with the 
+results.
+<4> Specifies the variable we want to predict with the {classanalysis}.
+<5> Specifies the proportion of data that is used for training the model. In 
+this example we use 10% of the source data. The training data is randomly 
+selected and provide predictions from the remaining 90%. These percentages are 
+approximate. For most of the datasets, even a small proportion of the training 
+data (2-5%) results in good predictions.
+<6> Specifies fields to be excluded from the analysis. It is recommended to 
+exclude fields that either contain erroneous data or describe the 
+`dependent_variable`.
+<7> Specifies a memory limit for the job. If the job requires more than this 
+amount of memory, it fails to start. This makes it possible to prevent job 
+execution if the available memory on the node is limited.
+
+
+The API returns the following response:
+
+[source,console-result]
+--------------------------------------------------  
+{
+  "id" : "model-delayed-flights",
+  "source" : {
+    "index" : [
+      "kibana_sample_data_flights"
+    ],
+    "query" : {
+      "range" : {
+        "DistanceKilometers" : {
+          "gt" : 0
+        }
+      }
+    }
+  },
+  "dest" : {
+    "index" : "df-flight-results",
+    "results_field" : "ml"
+  },
+  "analysis" : {
+    "classification" : {
+      "dependent_variable" : "FlightDelay",
+      "num_top_classes" : 2,
+      "prediction_field_name" : "FlightDelay_prediction",
+      "training_percent" : 70.0
+    }
+  },
+  "analyzed_fields" : {
+    "includes" : [ ],
+    "excludes" : [
+      "Cancelled",
+      "FlightDelayMin",
+      "FlightDelayType"
+    ]
+  },
+  "model_memory_limit" : "100mb",
+  "create_time" : 1574165518647,
+  "version" : "7.5.0",
+  "allow_lazy_start" : false
+}
+--------------------------------------------------
+--
+
+. Start the job.
++
+--
+Use the {ref}/start-dfanalytics.html[start {dfanalytics-jobs}] API to start the 
+job. It will stop automatically when the analysis is complete, you don't need to 
+stop it manually.
+
+[source,console]
+--------------------------------------------------
+POST _ml/data_frame/analytics/model-delayed-flights/_start
+--------------------------------------------------
+// TEST[skip:TBD]
+
+
+The job takes a few minutes to run. Runtime depends on the local hardware and 
+also on the number of documents and fields that analyzed. The more fields and 
```

Comment:
`... documents and fields that analyzed`

How about changing to "that are analyzed" or similar?